packages/relay/src/lib/eth.ts

```diff
@@ -1860,7 +1861,7 @@ export class EthImpl implements Eth {
   static isArrayNonEmpty(input: any): boolean {
     return Array.isArray(input) && input.length > 0;
   }
-
+  0x297a74308b60a564d6d2a4921ee19c92505430857ea6737b1e6c2fb3ad62ddc571fbfb9d0531ec2babd2c7a118b86b0b
```

Comment:
Bad merge, should remove

Reply:
True, something left from the rebase i guess. Will remove